### DISCUSSION TITLE
How can associate any items to the same taxonomy term?

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#17074] [Q&A] [answered]
If I have:
- an item A of type ContentTypeA
- an item B of type ContentTypeB
- a taxonomy T containing the terms T1 and T2
Is it possible to associate A and B to the same term of the taxonomy T?
For instante I'd like to to associate both A and B to T1.
If yes, how could I do it?

**Comment by Piedone** [comment, marked as answer]
Yes, you just need to add a Taxonomy Field to both types, and select the same Taxonomy under them. Then, when editing the content items, you'll be able to select the same terms, too.
